infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2Infrastructure.java

```diff
@@ -217,12 +217,13 @@ public void acquireNode() {
 
         Set<String> instancesIds = Sets.newHashSet();
 
+        // MAC addresses not yet managed
```

Comment:
Removed in https://github.com/vinseon/nodesource-addons/commit/ac0f9823268618d9e9dd5b1a20c4559b94364591